### PR TITLE
[INTERNAL] Upgrade devDependencies / Update CI matrix

### PR DESCRIPTION
**Pull request by matz3**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Enforce public npm registry
 registry = https://registry.npmjs.org/
+lockfile-version=3
```
Comment (flovogt) [inline .npmrc:3]: missing last empty line

**Review by flovogt — CHANGES_REQUESTED** [review]
this change is a breaking change, isn't? so we will release a new major version=

File: .npmrc (hunk 1 above)
Comment (matz3) [reply .npmrc:3]: File has been copied from https://github.com/SAP/ui5-fs/blob/main/.npmrc

**Comment by matz3** [comment]
Yes, it is breaking. As per node-semver, the next version should be `0.12.0`. I just wanted to finally upgrade the devDepenencies, but I don't really see a need of releasing a new version immediately as it does not have any impact on the shipped package itself.

File: .npmrc (hunk 1 above)
Comment (flovogt) [reply .npmrc:3]: Ack

**Review by flovogt — APPROVED** [review]

**Pull request merged**